constructor 3.13.0 (with rattler-build, pixi)

bollwyvl opened the pull request:
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

references:
- alternative to (and includes) #87 
- alternative to (and includes) #89
- ~~fixes #91 with a (temporary) script~~
  - > was fixed upstream

changes:
- [x] use `rattler-build`
- [x] use `pixi` 
- [x] use `shellcheck` (in `unix`)

regro-cf-autotick-bot committed:
updated v3.11.2

regro-cf-autotick-bot committed:
MNT: Re-rendered with conda-build 25.1.2, conda-smithy 3.46.1, and conda-forge-pinning 2025.03.07.20.58.59

bollwyvl committed:
try rattler-build

bollwyvl committed:
MNT: Re-rendered with conda-build 24.11.2, conda-smithy 3.47.0, and conda-forge-pinning 2025.03.11.09.37.43

bollwyvl commented:
@conda-forge-admin please rerender

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/recipe.yaml```) and found it was in an excellent condition.

bollwyvl commented:
Some perf numbers:

| what | screen |
|-|-|
| #87 | ![image](https://github.com/user-attachments/assets/810942ab-3ca3-48f3-8383-b2f4c1b1cc50)
| #88 | ![image](https://github.com/user-attachments/assets/e58a802f-1eac-4f50-abac-debf9cd0bc60)

bollwyvl commented:
@conda-forge/constructor ready for review

jaimergp commented:
Thanks! I prefer we handle the conversion separate from a new release, so I merged the other one. I'll keep this open though.

bollwyvl committed:
merge upstream

bollwyvl committed:
bump build number

bollwyvl commented:
@conda-forge-admin please rerender

bollwyvl commented:
@conda-forge/constructor ready for review

bollwyvl committed:
restore one magic comment for IDEs

regro-cf-autotick-bot committed:
updated v3.11.3

bollwyvl committed:
3.11.3

bollwyvl committed:
Merge remote-tracking branch 'origin/3.11.2_h4898c3' into 3.11.2_h4898c3

bollwyvl commented:
@conda-forge-admin please rerender

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-webservice.

I tried to rerender for you, but it looks like there was nothing to do.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/14008913380](https://github.com/conda-forge/conda-forge-webservices/actions/runs/14008913380). Examine the logs at this URL for more detail.</sub>

bollwyvl commented:
@conda-forge/constructor ready for review

bollwyvl committed:
update from upstream

bollwyvl committed:
MNT: Re-rendered with conda-build 25.5.0, conda-smithy 3.51.1, and conda-forge-pinning 2025.07.21.12.24.54

bollwyvl committed:
update test

bollwyvl commented:
@conda-forge-admin please rerender

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-webservice.

I tried to rerender for you, but it looks like there was nothing to do.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/16423944643](https://github.com/conda-forge/conda-forge-webservices/actions/runs/16423944643). Examine the logs at this URL for more detail.</sub>

bollwyvl committed:
ensure and test schema

bollwyvl committed:
avoid windows perms issues with shellcheck

bollwyvl committed:
use SP_DIR because windows

bollwyvl commented:
@conda-forge/constructor also fixed #91 :dotted_line_face:

bollwyvl committed:
use early, explicit schema file name test instead of import test

bollwyvl committed:
constructor 3.12.1

bollwyvl commented:
@conda-forge/constructor ready for review

jjerphan reviewed (approved):
Thank you, @bollwyvl.

bollwyvl committed:
constructor 3.12.2

bollwyvl committed:
constructor 3.13.0

bollwyvl committed:
MNT: Re-rendered with conda-smithy 3.52.3 and conda-forge-pinning 2025.11.04.13.36.45

Other tools:
- conda-build 25.9.0
- rattler-build 0.48.1
- rattler-build-conda-compat 1.4.5

bollwyvl committed:
merge origin

bollwyvl committed:
merge upstream

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-webservice.

I tried to rerender for you, but it looks like there was nothing to do.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/19073336252](https://github.com/conda-forge/conda-forge-webservices/actions/runs/19073336252). Examine the logs at this URL for more detail.</sub>

jaimergp committed:
Merge branch 'main' of github.com:conda-forge/constructor-feedstock into 3.11.2_h4898c3

MNT: Re-rendered with conda-smithy 2026.6.14 and conda-forge-pinning 2026.06.15.02.01.49

Other tools:
- conda-build 26.5.0
- rattler-build 0.66.1
- rattler-build-conda-compat 1.4.15

jaimergp committed:
Bump build number

jaimergp committed:
PYTHONHASHSEED="0"

jaimergp committed:
Move Windows to GHA

jaimergp committed:
no truststore

jaimergp committed:
legacy certs?

jaimergp committed:
Back to AZP
